APIURLS.md

```diff
@@ -0,0 +1,110 @@
+Admin Features:
+    Manage Project:

```

Comment:
Missing the the `Project Setting` function related API, basically you need and Patch API